nvflare/private/fed/app/client/worker_process.py

```diff
@@ -53,6 +54,12 @@ def main():
     parser = argparse.ArgumentParser()
     parser.add_argument("--workspace", "-m", type=str, help="WORKSPACE folder", required=True)
     parser.add_argument("--startup", "-w", type=str, help="startup folder", required=True)
+    parser.add_argument("--token", "-t", type=str, help="token", required=True)
```

Comment:
why we adding additional argument, is this needed for fix racing condition. We are in code freeze, i.e. only changes needed to relate to critical fixes

Reply:
thank Yuhong for explaining to me that the change is actually to replace token_file, which is also a race condition (share processed can access the same token file)